apis/server/utils.go

```diff
@@ -73,6 +75,21 @@ func writeLogStream(ctx context.Context, w io.Writer, tty bool, opt *types.Conta
 			}
 
 			logLine := msg.Line
+
+			if opt.Details {
+				var ss []string
+				for k, v := range msg.Attrs {
+					ss = append(ss, k+"="+v)
+				}
+				// keep the log attrs sorted
+				sort.Slice(ss, func(i, j int) bool {
+					keyI := strings.Split(ss[i], "=")
+					keyJ := strings.Split(ss[j], "=")
+					return keyI[0] < keyJ[0]
+				})
+				logLine = append([]byte(strings.Join(ss, ",")+" "), logLine...)
```

Comment:
There is a redundant space before log string if msg.Attrs is empty.
change the condition to `if opt.Details && len(msg.Attrs) != 0`

Reply:
done.